docs/content/documentation/getting-started/configuration.md

```diff
@@ -100,6 +100,12 @@ taxonomies = []
 # content for `default_language`.
 build_search_index = false
 
+# When set to "false", Sitemap.xml is not generated (default: true)
+generate_sitemap = false
+
+# When set to "false", robots.txt is not generated (default: true)
+generate_robots_txt = false
```

Comment:
We use the default in the docs so set them to true

Reply:
My bad, I didn't look carefully at the docs. It should be correct by now.